src/odemis/dataio/test/tiff_test.py

```diff
@@ -2359,7 +2359,7 @@ def testConvertTFStoOdemisMD(self):
 
         # assert metadata
         md = {
-            model.MD_ACQ_DATE: 1699621755.0,
+            # model.MD_ACQ_DATE: 1699621755.0, breaks in CI?
```

Comment:
There is no reason it breaks on the CI but not in a "normal" computer. So it's worth inverstigating. Do you have logs on how it failed (what was the actual value of MD_ACQ_DATE)? As it's a float, maybe you would need to use `assertAlmostEqual`.

Reply:
@pieleric I believe it is a timezone issue. The timestamps are exactly 1hr apart. This is because the TFS metadata doesn't have any timezone information, just a Date, Time string. Therefore, it just uses the default timezone of wherever the code is running, which is utc in CI, and whatever the timezone is set for on the Odemis computer. I can explicitly set the timezone in the parser so the tests pass, but that might be confusing to users rather than seeing it in their local timezone. What do you think?


CI Test
```bash
======================================================================
FAIL: testConvertTFStoOdemisMD (tiff_test.TestTiffIO)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/odemis/odemis/src/odemis/dataio/test/tiff_test.py", line 2384, in testConvertTFStoOdemisMD
    self.assertEqual(value, data.content[0].metadata[key])
AssertionError: 1699621755.0 != 1699625355.0
```

Local Test
``` bash
odemis@odemis-sim:~/development/odemis/src/odemis/dataio/test$ python3 -m unittest tiff_test.TestTiffIO.testConvertTFStoOdemisMD
WARNING:root:Failed to load optimised functions, slow version will be used.
INFO:root:Failed to use the OME data (No OME XML data found), will use standard TIFF
timestamp: 1699621755.0
.
----------------------------------------------------------------------
Ran 1 test in 0.021s

OK
```

Explictly setting tz=utc
```bash

odemis@odemis-sim:~/development/odemis/src/odemis/dataio/test$ python3 -m unittest tiff_test.TestTiffIO.testConvertTFStoOdemisMD
WARNING:root:Failed to load optimised functions, slow version will be used.
INFO:root:Failed to use the OME data (No OME XML data found), will use standard TIFF
timestamp: 1699625355.0
timezone: UTC
.
----------------------------------------------------------------------
Ran 1 test in 0.009s

OK
```


Checking Timestamps manually
```python
In [1]: timestamp = 1699625355.0

In [2]: import datetime

In [3]: date = datetime.datetime.fromtimestamp(timestamp)

In [4]: date
Out[4]: datetime.datetime(2023, 11, 11, 1, 9, 15)

In [5]: timestamp2 = 1699621755.0

In [6]: date = datetime.datetime.fromtimestamp(timestamp2)

In [7]: date
Out[7]: datetime.datetime(2023, 11, 11, 0, 9, 15)

In [8]: timestamp - timestamp2
Out[8]: 3600.0

```

Reply:
From the data bar, it seems that the time is stored as the local time (as they match, and this image was acquired in Australia, so if the time was stored as UTC internally it would be a quite different value!). 
So I think your code in tiff.py does the right thing. We just need to find the right way to test it.
One solution is to not check MD_ACQ_DATE... but it's a little sad.
The alternative is to just explicitly encode the time as a string (in the test case) and convert it to the local time.

Reply:
Ok, I've push a fix to explicitly encode the string in the test case, so it now works (CI still failing on 20.04 due to unrelated speed test)